package.json

```diff
@@ -20,8 +21,7 @@
     "format:tsx": "prettier --list-different \"src/**/*.{js,jsx,ts,tsx}\"",
     "format:scss": "prettier --list-different --parser scss \"src/**/*.scss\"",
     "format:ci": "yarn run format:tsx && yarn run format:scss",
-    "start": "cross-env DISABLE_ESLINT_PLUGIN=true BROWSER=none PORT=8000 craco start",
```

Comment:
Can we consider having `start` alias `dev` for those users still used to the old way of launching? Or do you think that is not necessary?

Reply:
I think we can just use start instead of dev